src/main.rs

```diff
@@ -1,10 +1,13 @@
 fn main() -> eyre::Result<()> {
     // read env variables that were set in build script
     let bios_path = env!("BIOS_PATH");
-
+    let cargo_dir = env!("CARGO_MANIFEST_DIR");
+    let bootsplash_argument =  format!("menu=on,splash={cargo_dir}/src/assets/bootsplash.bmp,splash-time=1000");
```

Comment:
```suggestion
    let bootsplash_path = format!({cargo_dir}/src/assets/bootsplash.bmp);
    let boot_arg = format!("menu=on,splash={bootsplash_path},splash-time=1000");
```

Reply:
you can also not create boot_arg variable and just have the format inside the `arg()` fn